src/main/kotlin/no/nav/k9/los/nyoppgavestyring/query/dto/felter/Verdiforklaring.kt

```diff
@@ -2,6 +2,7 @@ package no.nav.k9.los.nyoppgavestyring.query.dto.felter
 
 class Verdiforklaring(
     val verdi: String,
-    val visningsnavn: String
+    val visningsnavn: String,
+    val sekundærvalg: Boolean
```

Comment:
ønsker meg en forklarende kommentar på dette feltet, kanskje? Hvordan blir det at noe ikke er en favoritt til et sekundært valg? Hva er et sekundært valg for noe?

Reply:
Det er mindre brukte kodeverksverdier. Dette tilsvarer "avansert" og knyttet til visningen - der sekundærvalg vises separat nederst i dropdown. Det virket litt feil å lagre det som sekundærvalg i db, men er er åpen for forslag

Reply:
hm, første refleks er at "mindreBrukt" i så fall er en forbedring

Reply:
evt "sorterTilSlutt"/"sorterNederst"

Reply:
Hvorfor trenger vi en inversform i verdiforklaring? Kan det ikke bare være "favoritt" her også?